test/utils/ConsolidatedEvents.sol

```diff
@@ -6,16 +6,30 @@ import {IAccount} from "../../src/interfaces/IAccount.sol";
 /// utility contract for *testing* events. consolidates all events into one contract
 
 contract ConsolidatedEvents {
+    /*//////////////////////////////////////////////////////////////
+                                IFACTORY
+    //////////////////////////////////////////////////////////////*/
+
+    event NewAccount(
+        address indexed creator, address indexed account, bytes32 version
+    );
+
+    event AccountImplementationUpgraded(address implementation);
+
+    event EventsUpgraded(address events);
```

Comment:
**Suggestion:** I think this can be removed as it is no longer used?

Reply:
Right, will remove.